### ISSUE TITLE
git view file history does not work

### ISSUE THREAD

**Issue by zeroli** [DonJayamanne/gitHistoryVSCode#383]
git file history menu does not work
linux platform, vs code 

the git log shows below error, when specifically git view one file history from popup menu:

git log --full-history --pretty=oneline --date-order --decorate=full --skip=0 --max-count=100 master --follow -- ../../../../../home/XXX/test-git/a/b/c.cc  (completed in 0.376s)
fatal: ../../../../../home/XXX/test-git/a/b/c.cc: '../../../../../home/XXX/test-git/a/b/c.cc' is outside repository

My home is mounted to /n/home/dev/XXX/

**Comment by weslowsk** [comment]
I get the same error on a Linux Mint 18 platform.

**Comment by zhanglid** [comment]
Me too
fatal: '../../../../../../../xxx' is outside repository

**Comment by ole1986** [comment]
Is the project containing your repo opened with a *.code-workspace file? Or simply opened as directory?

**Comment by zhanglid** [comment]
I opened it as directory. But I use viscose-remote

**Comment by ole1986** [comment]
It would be helpful to know at least some more details about the repository and its relative path to the file you are trying to open.

**Comment by ole1986** [comment]
Or even better, please check the output from "Git History" and paste here.
Important line starts with "Server running on..." once  you open the file.
E.g.

```
[...]
Server running on http://localhost:53315/?_&id=XXX&branchName=master&file=%2FUsers%2Fole%2FDocuments%2Fpdf-php%2Fsrc%2Finclude%2FTTF.php&theme=&branchSelection=1&locale=de_DE
[...]
```

**Comment by ole1986** [comment]
Is this still an issue with version 0.5.0?

**Comment by troy351** [comment]
Still exists in v0.6.5, seems only happened on Linux and works fine on Windows

**Comment by ole1986** [comment]
This should be fixed together with #565 and #556
